src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -183,6 +184,10 @@ def parse_performance_features(log_lines, strategies, arguments):
       'timeout_count': 0,
   }
 
+  # Extract strategy selection method.
+  stats['strategy_selection_method'] = environment.get_value(
```

Comment:
Could you use a default value of 'default' here rather than worrying about setting it in the default case? It feels a bit safer to me than what we're currently doing.